Documentation/applications/nsh/commands.rst

```diff
@@ -736,7 +736,7 @@ at <link> for the existing file or directory, <target>. This
 implementation is simplified for use with NuttX in these ways:
 
   -  Links may be created only within the NuttX top-level, `pseudo
-     file system <NuttxUserGuide.html#FileSystemOverview>`__. No
```

Comment:
@xiaoxiang781216 unfortunately we did not use `ref` here like we should have so this will break the link.  We might be able to just rename the referenced file to work around it for now.  I think there is a documentation ticket to go and update to use refs.
![image](https://user-images.githubusercontent.com/173245/96539429-dc241280-124f-11eb-9e53-5454e663f976.png)

Reply:
Yes, this patch just to ensure the whole code base use 'NuttX' term. The broken link need to address in the future patch. Since many reference point to the document not convert from htm to rst yet, I am considering to fix the broken link once all referenced document move to git.

Reply:
I just submitted #2045 which we can apply later to fix this.